codacy/requirements-dev.yaml

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: minio
-  version: 2.5.18
+  version: 5.0.33
```

Comment:
requirements.yaml had already been updated. What seems to matter most is the `codacy/values.yaml` file `minio.image.tag` value.